newsletters/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from django.shortcuts import render
+from django.views.generic import TemplateView
+from . import views
+
+urlpatterns = [
+	path('', views.Index.as_view(template_name='newsletters/index.html'), name='newsletters_index'),
+	path('create', views.Create.as_view(template_name='newsletters/create.html'), name='newsletters_create'),
+	path('<int:pk>', views.Detail.as_view(template_name='newsletters/detail.html'), name='newsletters_detail'),
+	path('edit/<int:pk>', views.Update.as_view(template_name='newsletters/update.html'), name='newsletters_update'),
+	path('delete/<int:pk>', views.Delete.as_view(template_name='newsletters/delete.html'), name='newsletters_delete'),
+	path('email-version/<int:pk>', views.Detail.as_view(template_name='newsletters/email-version.html'), name='newsletters_email'),
```

Comment:
Is this for linking to in emails?

Reply:
Not sure how it can be done - currently the email version page displays the newsletter without the topbar and I tried saving it in Firefox (with Save as HTML) with some success - not sure whether in the email we can cheat and just iframe to it or if we have to paste in the HTML as I tried

Reply:
We could make a command to generate a newsletter, that essentially renders and saves the output to a file?

In that case `email-version` wouldn't be necessary as a URL, unless we want to do that "Not looking right? View in browser" link, which does seem like a good idea